### ISSUE TITLE
Fix translation de

### ISSUE THREAD

**Issue by Yiudorai** [NECROBOTIO/NecroBot#3759]
Hi im german and i found some mistakes in your translation thats correct but show in dose like this `PokÃ©ball` but it must look like this `Pokeball`.

Also you got ä, ö and ü that cant show in Dos and in germany we use ae for ä, oe for ö and ue for ü if it cant show.

Please change the german translation in next version whit following thinks:
`é = e
ä = ae
ö = oe
ü = ue`

So it will look better for us ;)

**Comment by DrLippe** [comment]
"18.20m entfernt | mit Pokéball (17 verbleibend). | 210 EP erhalten" - all fine

**Comment by Yiudorai** [comment]
@DrLippe than you got any changes on your Windows because it looks like "PokÃ©ball" if you got normal Windows.

**Comment by Plotin** [comment]
@Yiudorai Nope, it is on your end, my Windows is plain Windows 10 and everything looks fine, with a german Windows
